admin/lib/solidus_admin/system/loaders/host_overridable_constant.rb

```diff
@@ -60,7 +60,7 @@ def override_constant(component, namespace)
             "#{application_name}::SolidusAdmin::#{component_segment(component, namespace).classify}".constantize
           end
 
-          # "components.foo.component" => "foo/component"
+          # "components/foo/component" => "foo/component"
```

Comment:
This is wrong, as we didn't change the namespace separator in the container, only in the helper.